settings/base.py

```diff
@@ -1,13 +1,11 @@
 """
 Django settings for miete project.
 
-Generated by 'django-admin startproject' using Django 1.8.4.
-
 For more information on this file, see
-https://docs.djangoproject.com/en/1.8/topics/settings/
+https://docs.djangoproject.com/en/1.9/topics/settings/
```

Comment:
Ich hatte gedacht, wir swollen die neuste Version, weil in `requirements`.txt` `django>=1.8` steht. Habe das jetzt mal korrigiert